AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3633,6 +3633,7 @@ bmwfaq.org#?#.Notices[-ext-has='a[href="//www.bmwfaq.org/help/cookies"]']
 ! JS rules
 !
 eluban.pl#%#document.cookie="rodopop=1; path=/;";
+usnews.com#%#document.cookie="gdpr_agreed=4";
```

Comment:
It seems can be blocked by `.modal-window`. I don't see other popups with this class.

Reply:
Social sharing is not blocked at least in Photo section.